op-batcher/batcher/config.go

```diff
@@ -31,6 +31,15 @@ type Config struct {
 	From         common.Address
 
 	TxManagerConfig txmgr.Config
+	NetworkTimeout  time.Duration
+
+	// TxManagerTimeout is the maximum amount of time
+	// the driver should wait for the [txmgr] to send a transaction.
+	TxManagerTimeout time.Duration
```

Comment:
This is a good thing to parameterize, but a little out of scope for this PR which should be focused on where the transaction is created. Also given that we have the op-propser which uses the txmanager with the same flags, we will want to create a common set of flags inside the txmanager.

Reply:
I can change this in the proposer but it seems like _that_ would be out of the scope of this pr. lmk